Signal/ConversationView/Components/CVComponentSystemMessage.swift

```diff
@@ -57,13 +57,7 @@ public class CVComponentSystemMessage: CVComponentBase, CVRootComponent {
 
     private var innerVStackConfig: CVStackViewConfig {
 
-        let layoutMargins: UIEdgeInsets
-        if itemModel.itemViewState.isFirstInCluster {
```

Comment:
I think that removing this will lead to other UI issues. We intentionally add that top padding for those `isFirstInCluster` items, to visually separate those clusters.